File: JavaScript Language/JavaScript Language/Function.js
    // javascript - function 
    function println(msg){
        console.log(msg);
    }
    // calling function
    println("Hello World - using println function");

    // function to print factorila using recursion
    function factorial(num){
        if(num==0) return 1;
        else{
            return num*factorial(num-1);
        }
    }
    println("factorial of 5 = "+factorial(5));

    function display(msg='No Message'){
        console.log(msg);
    }

    display();
    display('Hi User');

File: R/R Basic/Function.r
# R - Function

print('Hello World')

sum <- function(a,b){
    print(a+b)
}
sum(1,2)

table <- function(num){
    for(i in 0:5){
        print(i*num)
    }
    print('')
}
table(10)

test <- function(a=10){
    print(a)
}
test(a=100)

class(test)